org.scala.tools.eclipse.search/META-INF/MANIFEST.MF

```diff
@@ -27,7 +27,10 @@ Require-Bundle:
  org.scala-lang.scala-compiler;bundle-version="[2.11,2.12)",
  org.scala-lang.scala-reflect;bundle-version="[2.11,2.12)",
  org.scala-ide.sdt.core;bundle-version="[4.0.0,5.0.0)",
- org.eclipse.search
+ org.eclipse.search,
+ org.eclipse.core.databinding.beans;bundle-version="1.2.200",
+ org.eclipse.jface.databinding;bundle-version="1.6.200",
+ org.eclipse.core.databinding.property;bundle-version="1.4.200"
```

Comment:
Please remove the bundle versions here. If none is used, the latest version available will be used. That should make updates to new Eclipse versions easier/possible.